app/controllers/game.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Module dependencies.
+ */
+
+import mongoose from 'mongoose';
+
+const Game = mongoose.model('Game');
+
+const gameResult = (req, res) => {
+  let gameId = req.params.id;
+  gameId = parseInt(gameId, 10);
+  const { gameStarter, gameWinner, players } = req.body;
+  if (!gameStarter || !players) {
+    return res.status(401).json({ message: 'Game have not started' });
```

Comment:
Fix the type here

Reply:
I don't think `gameStarter` is necessary here. All that is needed is the `gameId` & `players`. What value will saving the `gameStarter` bring to the application

Reply:
Noted